auditing/logfile.go

```diff
@@ -0,0 +1,36 @@
+package auditing
+
+import (
+    "bytes"
+    "log"
+    "os"
+)
+// Log logs information using the standard Go logger package
+func Log(logMessage string) {
```

Comment:
A future improvement might be to use a premade logging framework such as https://github.com/sirupsen/logrus